src/midi.cpp

```diff
@@ -114,7 +157,75 @@ static void HandleControlChange(byte channel, byte data1, byte data2) {
 			velocityToLfo = 0;
 		}
 	} // velocity -> lfo depth3
+	else if ((channel == 16) && (toolMode) && (data1 == 88)) {
+		if (data2) {
+			EEPROM.update(3990, 1);
+			pwLimit = 1;
+		} else {
+			EEPROM.update(3990, 0);
+			pwLimit = 0;
+		}
+	} // pwLimit
+
+	else if ((channel == 16) && (toolMode) && (data1 == 89)) {
+		volumeChanged = true;
+		volume = data2;
+		if ((volume > 15) || (volume < 1)) {
+			volume = 15;
+		}
+		EEPROM.update(3991, volume);
+	} // master volume
+
+	else if ((channel == 16) && (toolMode) && (data1 == 90)) {
+		if (data2 < 16) {
+			EEPROM.update(3998, data2 + 1);
+			masterChannel = data2 + 1;
+		}
+	} // master input channel
 
+	else if ((channel == 16) && (toolMode) && (data1 == 91)) {
+		if (data2 < 16) {
+			EEPROM.update(3997, data2 + 1);
+			masterChannelOut = data2 + 1;
+		}
+	} // master output channel
+	else if ((channel == 16) && (toolMode) && (data1 == 92)) {
+		if (data2) {
+			EEPROM.update(3996, 1);
+			sendLfo = 1;
+		} else {
+			EEPROM.update(3996, 0);
+			sendLfo = 0;
+		}
+	} // lfo transmits CC
+	else if ((channel == 16) && (toolMode) && (data1 == 94)) {
```

Comment:
all these parentheses... I'd rather remove them but i'm too lazy to write all these suggestions. It's fine if you're too lazy, too :)